uint64 timestamp
float64 quat_w
float64 quat_x
float64 quat_y
float64 quat_z
float64 accel_x
float64 accel_y
float64 accel_z
